site/source/docs/compiling/Modularized-Output.rst

```diff
@@ -108,6 +109,18 @@ found on the module object are instead exported directly.  For example:
   await init({ print: myPrint });
   _nativeMethod();
 
+Limitations
+-----------
+
+Some major features still do not work in this mode:
+
+* Internal usage (e.g. usage within EM_JS / JS libary code) of the `Module`
+  object does not work.  This is because symbols are exported directly using
+  ES6 module syntax rathar than using a global `Module` object.
+
+* `ccall`/`cwrap`, these depend on the internal `Module` object.
```

Comment:
This seems more like a bug than a true limitation - perhaps open an issue for it instead (maybe link from here)?

Reply:
I'm planning on listing a lot of limitation here, and they can all likely be considered bugs.

Perhaps if I called this section "Current Limitations"?